presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/IcebergTableProperties.java

```diff
@@ -26,13 +26,15 @@
 
 import static com.google.common.collect.ImmutableList.toImmutableList;
 import static io.prestosql.spi.session.PropertyMetadata.enumProperty;
+import static io.prestosql.spi.session.PropertyMetadata.stringProperty;
 import static io.prestosql.spi.type.VarcharType.VARCHAR;
 import static java.util.Locale.ENGLISH;
 
 public class IcebergTableProperties
 {
     public static final String FILE_FORMAT_PROPERTY = "format";
     public static final String PARTITIONING_PROPERTY = "partitioning";
+    public static final String EXTERNAL_LOCATION_PROPERTY = "external_location";
```

Comment:
Let's name this `location` to match the upcoming change to the Hive connector in #1282. This also makes more sense, as we are simply specifying the location and not creating an external table (which doesn't exist as a concept in Iceberg).